packages/database/src/repository/User.ts

```diff
@@ -239,9 +241,14 @@ export function UserRepository(database?: DataSource): Repository {
   async function findOneByUsername(username: string): Promise<User> {
     return BaseRepo.createQueryBuilder('user')
       .where('user.username = :username', { username })
-      .leftJoinAndSelect('user.modulesDone', 'modulesDone')
-      .leftJoinAndSelect('user.modulesDoing', 'modulesDoing')
       .getOneOrFail()
+      .then(async (user) => {
+        return UserRepository(db)
+          .loadRelations(user, allRelations)
+          .then(() => {
+            return user
+          })
+      })
```

Comment:
this change would break the test in `user/initialize.test.ts`,  feel free to remove 'Can find same user (without relations)'